src/App/Fossa/Init/.fossa.yml

```diff
@@ -0,0 +1,69 @@
+# # fossa-cli configuration file.
+# #
+# # All fields are optional except 'version' field.
+# #
+# #  To learn more:
+# #   - https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-yml.md
+# #
+# #  If you need assistance, or would like to file a defect, contact support at:
+# #   - https://support.fossa.com/
+# #
+# # -----
+
+version: 3
+# server: https://app.fossa.com
+# apiKey: a1b2c3
+#
+# project:
+#   id: github.com/fossas/fossa-cli
```

Comment:
IMO, each entry in this file should have a short comment explaining what it's for.

E.g.:

```
# # Your FOSSA API key. You can obtain this by going to the settings => Integrations => API section of the FOSSA app. This will be found at `<server>/account/settings/integrations/api_tokens`.
```